src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,6 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50</TargetGroup>
+    </Project>
+    <Project Include="System.Private.Xml.csproj">
+      <TargetGroup>netcore50aot</TargetGroup>
+    </Project>
```

Comment:
Do you know what code is different under that @ifdef? It looks like you removed a lot of those and it would be interesting to try and eliminate any unnecessary builds.